src/typescript.ts

```diff
@@ -20,7 +20,7 @@ import * as ts from 'typescript';
 // Note, this import depends on a genrule copying the .d.ts file to this package
 import * as ts24 from './typescript-2.4';
```

Comment:
I think we now need to add a test case for ts25 as well

Reply:
Done.